cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,25 @@
+class PaymentStepPage {
+  private payByBankWireBtn: string;
+  private proceedToCheckoutBtn: string;
+  private confirmationMessage_: string;
+
+  constructor() {
+    this.payByBankWireBtn = "[title='Pay by bank wire']";
+    this.proceedToCheckoutBtn = ".cart_navigation button";
+    this.confirmationMessage_ = "#center_column > div > p > strong";
+  }
+
+  public clickPayByBankWireBtn(): void {
+    cy.get(this.payByBankWireBtn).click();
+  }
+
+  public clickProceedToCheckout(): void {
+    cy.get(this.proceedToCheckoutBtn).click();
+  }
+
+  public get confirmationMessage(): string {
+    return this.confirmationMessage_;
+  }
+}
+
+export default new PaymentStepPage();
```

Comment:
not sure about this approach, I prefer the one suggested by the guide, export the class and then have a page/index.ts so you can import from the page index in the buy-shirt.cy.ts file